spec/models/course/assessment/assessment_ability_spec.rb

```diff
@@ -55,6 +61,19 @@
       it { is_expected.not_to be_able_to(:update, coursemate_attempting_submission) }
       it { is_expected.not_to be_able_to(:read, coursemate_submitted_submission) }
 
+      it do
+        is_expected.to be_able_to(:destroy_attachment,
+                                  get_text_response_answer_for attempting_submission)
```

Comment:
better add brackets for non DSL methods: `get_text_response_answer_for(attempting_submission)`